.gitignore

```diff
@@ -17,13 +17,11 @@ vendor/
 tsconfig.tsbuildinfo
 __debug_bin*
 
-# Yarn
-.yarn/
-!.yarn/patches
-!.yarn/releases
-!.yarn/plugins
-!.yarn/sdks
-!.yarn/versions
+# yarn
+.yarn/cache
+.yarn/unplugged
+.yarn/build-state.yml
+.yarn/install-state.gz
```

Comment:
@jackw 

with this change I now have 

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
        .yarn/.cache/
        .yarn/sdks/
```

Reply:
but seem like I can delete those, thought sdks would come back after `yarn dlx @yarnpkg/sdks vscode`

Reply:
@torkelo pretty sure we don't [use sdks](https://yarnpkg.com/getting-started/editor-sdks) any more with yarn berry as we stopped using PnP a while back. Do you have them referencing your vscode settings somewhere? 🤔

Reply:
[This PR that switched us off PnP](https://github.com/grafana/grafana/pull/79947) removed the sdks. The .cache directory should be fine to delete as I think that's where PnP stores the .zip files.

Reply:
If we think it's gonna cause widespread bother we can add them as ignored files.